app/src/main/java/com/framgia/forder/screen/createProduct/CreateProductContract.java

```diff
@@ -11,11 +16,29 @@
      * View.
      */
     interface ViewModel extends BaseViewModel<Presenter> {
+        void onGetCategoriesSuccess(List<Category> categories);
+
+        void onGetCategoriesError(BaseException error);
+
+        void onInputNameError();
+
+        void onInputDescriptionError();
+
+        void setUriImage(Uri uriImage);
+
+        void onRegisterProductSuccess();
+
+        void RegisterProductError(BaseException error);
```

Comment:
ERROR: [MethodNamingConventions] Method names should not start with capital letters
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#MethodNamingConventions